src/pages/FirstPage.jsx

```diff
@@ -11,48 +11,87 @@ const FirstPage = () => {
         <SubTitle>어르신 돌보미</SubTitle>
         <Title>케어코디</Title>
       </CardContainer>
-      <LandingButton onClick={() => setStep({ ...step, number: 0 })}>신청하기</LandingButton>
+      <ButtonContainer>
+        <LandingButton onClick={() => setStep({ ...step, number: 0 })}>신청하기</LandingButton>
+      </ButtonContainer>
     </Container>
   );
 };
 
 const Container = styled.div`
+  overflow-y: scroll;
   display: flex;
   align-items: center;
   justify-content: center;
   flex-direction: column;
-  width: 768px;
+  max-width: 768px;
+  min-width: 360px;
   height: 100vh;
-  margin: 0 auto;
-  font-weight: 600;
 `;
 
 const CardContainer = styled.div`
   display: flex;
   align-items: center;
   justify-content: center;
   flex-direction: column;
-  font-size: 2rem;
-  line-height: 1rem;
 `;
 
-const SubTitle = styled.h1`
-  font-size: 3rem;
+const SubTitle = styled.h5`
+  width: 210px;
+  height: 52px;
+  left: 0;
+  top: 0;
+  margin: 0;
+  font-size: 32px;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 52px;
+  text-align: center;
   color: ${({ theme }) => theme.fontColor};
 `;
 
-const Title = styled.h2`
+const Title = styled.h4`
+  width: 130px;
+  height: 52px;
+  margin: 0;
+  top: 52px;
+  left: 33px;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+  font-style: bold;
   color: ${({ theme }) => theme.buttonMain.backgroundColor};
-  padding-bottom: 40px;
 `;
 
-const LandingButton = styled.button`
+const ButtonContainer = styled.button`
   color: ${({ theme }) => theme.buttonMain};
```

Comment:
```suggestion
  ${({ theme }) => theme.buttonMain};
```
buttonMain의 style을 적용하시려면 위와 같이 작성하시고, color만 적용하시려면 아래와 같이 작성하시면 됩니다!
```js
color: ${({ theme }) => theme.buttonMain.color};
```